src/tests/stream-tcp-list.c

```diff
@@ -686,6 +686,7 @@ static int StreamTcpReassembleTest30 (void)
     OVERLAP_STEP(3, "BBB", 3, "\0\0BBB", 5);
     OVERLAP_STEP(1, "AA", 2, "AABBB", 5);
     OVERLAP_END;
+    PASS;
```

Comment:
As Shivani has indicated, some of these functions and macros already invoke the FAIL/PASS API (like we can see with `OVERLAP_END`- https://github.com/OISF/suricata/blob/master/src/tests/stream-tcp-list.c#L55-L58 )

So wherever we see that `OVERLAP_END` was called, we don't need to include a `PASS` call.